uint32 state_machine_id
---
bool is_enabled
